lib/secrets.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Secrets do
+  @moduledoc false
+
+  use Bitwise
+
+  def secret_add(num) do
+    fn arg -> arg + num end
```

Comment:
Please read about function captures: https://hexdocs.pm/elixir/master/Kernel.SpecialForms.html#&/1

```suggestion
    & &1 + num
```